### DISCUSSION TITLE
Version for mods, with interactions mentios, send messages, commands and kick/ban/delete message funcitions

### DISCUSSION THREAD

**Discussion by Albrod** [baffler/Transparent-Twitch-Chat-Overlay#68] [Ideas]
I'm mod of several streams in a lot of them im playing with the streamer and this toll help me a lot to keep track of chat while playing but will be nice if i could also write to the chat with this, send poll and ban or delete messages from this place while playing maybe in a separated version

**Comment by baffler** [comment]
Should already be possible. In settings, click General and make sure "Allow Interaction" is checked. And under settings for "Chat", change the chat type to "Twitch Popout" (it's the combo box at the bottom next to the Save button). Once you put in the username there, click Save and then try to say something in the chat box, then it will ask you to login, and you should be able to do any mod actions now.

**Reply by Albrod** [reply]
thanks didn't see that option, the only thing i see its left behind are the poins no?
